client/cli/src/client_proxy.rs

```diff
@@ -58,7 +58,7 @@ use transaction_builder::encode_register_validator_script;
 
 const CLIENT_WALLET_MNEMONIC_FILE: &str = "client.mnemonic";
 const GAS_UNIT_PRICE: u64 = 0;
-const MAX_GAS_AMOUNT: u64 = 400_000;
+const MAX_GAS_AMOUNT: u64 = 600_000;
```

Comment:
just curious why the increase?

Reply:
It's because we merged publishing option and gas schedule together! Thus updating this config item will cost more gas because the config size gets larger.